javascript-carousel/main.js

```diff
@@ -0,0 +1,79 @@
+var imgList = [
+  'images/001.png',
+  'images/004.png',
+  'images/007.png',
+  'images/025.png',
+  'images/039.png'
+];
+
+var dotIndex = 0;
+
+var $chevronContainer = document.querySelector('.chevron-container');
+var $dotContainer = document.querySelector('.dot-container');
+var $faCircle = document.querySelectorAll('.fa-circle');
+var $picture = document.querySelector('.picture');
+
+var intervalID = null;
+intervalID = setInterval(rotateCarousel, 3000);
+
+$chevronContainer.addEventListener('click', function (event) {
+  if (event.target.matches('i')) {
+    clearInterval(intervalID);
+    if (event.target.className === 'fas fa-chevron-right') {
+      if (dotIndex === 4) {
```

Comment:
Instead of using a magic hard-coded number here, you should make use of the `length` of the array that contains the values you want to loop over.